### PR TITLE
fix repl test failed

### LINKED ISSUE #2020: Confirm SQLFlow parser supports unicode input
https://github.com/sql-machine-learning/sqlflow/pull/2004#pullrequestreview-376204845

### PR DESCRIPTION
**Pull request by Yancey0623**
fix #2022

### CODE REVIEW

File: pkg/sql/executor_ir.go [hunk 1]
```diff
@@ -61,11 +61,8 @@ func RunSQLProgram(sqlProgram string, modelDir string, session *pb.Session) *pip
 		}
 		err = runSQLProgram(wr, sqlProgram, db, modelDir, session)
 		if err != nil {
-			err = wr.Write(fmt.Errorf("runSQLProgram error: %v", err))
-			if err != pipe.ErrClosedPipe {
-				if err := wr.Write(err); err != nil {
```
Comment (Yancey0623) [inline pkg/sql/executor_ir.go:57]: The root reason is written `nil` into the pipe at L66, then cause step panic.

File: pkg/sql/executor_ir.go [hunk 2]
```diff
@@ -61,11 +61,8 @@ func RunSQLProgram(sqlProgram string, modelDir string, session *pb.Session) *pip
...
-					log.GetDefaultLogger().Errorf("runSQLProgram error(piping): %v", err)
-				}
+			if e := wr.Write(fmt.Errorf("runSQLProgram error: %v", err)); e != nil {
+				log.GetDefaultLogger().Errorf("runSQLProgram error(piping): %v", err)
```
Comment (typhoonzero) [inline pkg/sql/executor_ir.go]: This change may cause the client (like in Jupyter Notebook) will not know the error message if the Python program fails?

Comment (Yancey0623) [reply pkg/sql/executor_ir.go]: May not, L64 tried to write `fmt.Error` to the pipe.

File: pkg/sql/executor_ir.go (hunk 1 above)
Comment (tonyyang-svail) [reply pkg/sql/executor_ir.go:57]: My apology for introducing this on https://github.com/sql-machine-learning/sqlflow/pull/1983/files#diff-45b52a537a003bab57099e84c65c0985R64

**Comment by tonyyang-svail** [comment]
Hi @Yancey1989, could you please note why this PR is related to #2020? I am asking because this PR doesn't look like related to Unicode.

**Comment by Yancey0623** [comment]
> Hi @Yancey1989, could you please note why this PR is related to #2020?

Oh, I type the wrong issue No. , it should be #2022

File: pkg/sql/executor_ir.go [hunk 3]
```diff
@@ -53,10 +53,8 @@ func RunSQLProgram(sqlProgram string, modelDir string, session *pb.Session) *pip
 		}
 		err = runSQLProgram(wr, sqlProgram, db, modelDir, session)
 		if err != nil {
-			if err != pipe.ErrClosedPipe {
```
Comment (shendiaomo) [inline pkg/sql/executor_ir.go:56]: Can we remove this condition about `pipe.ErrClosedPipe`?

**Review by typhoonzero — APPROVED** [review]
LGTM++

**Pull request merged**